scripts/db_migrator.py

```diff
@@ -1030,6 +1059,7 @@ def common_migration_ops(self):
             log.log_notice("Asic Type: {}, Hwsku: {}".format(self.asic_type, self.hwsku))
 
         self.migrate_route_table()
+        self.migrate_sflow_table()
```

Comment:
What's the reason for doing this in common migration logic, and not in version specific migration?
Please keep in mind that common migration logic gets called always, irrespective of the which DB versions are current on new OS.

Reply:
Understood. I moved the logic under specific migration.